docs/install/harvester-configuration.md

```diff
@@ -515,6 +515,87 @@ install:
   data_disk: /dev/sdb
 ```
 
+### `install.harvester.storageClass.replicaCount`
```

Comment:
please use snake_case instead.

Reply:
this is an inherited design from k3os(harvester's original baseOS), and making it consistent with the existing doc format.